tools/buildsteps/windows/arm-uwp/BuildSetup.bat

```diff
@@ -0,0 +1,12 @@
+@ECHO OFF
+
+SET cmakeGenerator=Visual Studio 14 ARM
+SET TARGET_ARCHITECTURE=x86
```

Comment:
command `cmake.exe --build . --config "%buildconfig%"` produces appxbundle wich is ready to upload to win store, so no need any packaging. one big difference for UWP will be a bit different approach to build.
1. firstly have to build libkodi
2. build binary addons
3. copy addons to build kodi dir
4. final build (it will proper package all binary addons)